projects/plugins/jetpack/extensions/blocks/conversation/utils.js

```diff
@@ -14,6 +14,6 @@ export function getParticipantByLabel ( participants, participantLabel ) {
 	return part?.length ? part[ 0 ] : null;
 }
 
-export function cleanFormatStyle( html ) {
+export function getPlainText( html ) {
```

Comment:
Would it be an option to use `escapeHTML` from `@wordpress/escape-html` directly within that function, so it's applied to anything using `getPlainText`?

Reply:
yes, it could work. Do you mean an option through a function parameter?

```es
export function getPlainText( html, escape = false ) {
  // ...
}
```
?

Reply:
tackled here https://github.com/Automattic/jetpack/commit/6c384acf306527797ad57af29393df03b1337a08